htdocs/core/lib/project.lib.php

```diff
@@ -1116,8 +1120,11 @@ function print_projecttasks_array($db, $form, $socid, $projectsListId, $mytasks=
     	}
     	print_liste_field_titre($langs->trans("Status"),"","","","",'align="right"',$sortfield,$sortorder);
     	print "</tr>\n";
-		
-		while ($i < $num)
+
+		if ($limit == 0)
```

Comment:
If param limit is not provided or provided to 0 (meaning no limit), we must not force a limit.